From 37c4e82e26eac4a52d34339a8d2bae7311d59cd8 Mon Sep 17 00:00:00 2001
From: Marcus Comstedt <marcus@mc.pp.se>
Date: Tue, 29 Mar 2022 17:56:36 +0200
Subject: [PATCH] xre: Fix sandbox include

These includes are needed for any sandbox functionality, not just
GMP sandbox.
(Specifically, XRE_InitChildProcess needs mozilla::SandboxEarlyInit.)
---
 toolkit/xre/nsEmbedFunctions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolkit/xre/nsEmbedFunctions.cpp b/toolkit/xre/nsEmbedFunctions.cpp
index 4cb6ad9e0..08a69b394 100644
--- a/toolkit/xre/nsEmbedFunctions.cpp
+++ b/toolkit/xre/nsEmbedFunctions.cpp
@@ -90,7 +90,7 @@
 #include "mozilla/Preferences.h"
 #endif
 
-#if defined(XP_LINUX) && defined(MOZ_GMP_SANDBOX)
+#if defined(XP_LINUX) && defined(MOZ_SANDBOX)
 #include "mozilla/Sandbox.h"
 #include "mozilla/SandboxInfo.h"
 #endif
-- 
2.35.1

